### ISSUE TITLE
Entering in garbage input crashes the application

### ISSUE THREAD

**Issue by LarryGasik** [sunilumesh/DesignCrowdChallenge#4]
When i enter in a non-valid date for the application, it crashes the application. It should say something like, "That's not a valid date. Try again."

**Comment by sunilumesh** [comment]
I am unable to replicate this issue. When I enter garbage, I get the message 'Invalid value entered, unable to proceed'

**Comment by LarryGasik** [comment]
Right! I'm saying from a user perspective issue. I am giving you this as a challenge. I want you to figure out a way so that when I enter in garbage to the UI, It shouldn't say "Invalid value...." 

Instead, I want you to give me a chance to try again.
